test/include/auxkernels/GetElementalValueAux.h

```diff
@@ -0,0 +1,40 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef GETELEMENTALVALUEAUX_H
+#define GETELEMENTALVALUEAUX_H
+
+#include "AuxKernel.h"
+
+class GetElementalValueAux;
+
+template <>
+InputParameters validParams<GetElementalValueAux>();
+
+/**
+ * Tests the \c getElementalValue() function of \c MooseVariableFE.
+ */
+class GetElementalValueAux : public AuxKernel
+{
+public:
+  GetElementalValueAux(const InputParameters & params);
+
+  virtual ~GetElementalValueAux();
```

Comment:
I believe we do not need empty destructors any more.

Reply:
Correct, unless you are creating a new base class.